lib/adapters/REST/endpoints/app-signing-secret.ts

```diff
@@ -0,0 +1,51 @@
+import type { AxiosInstance } from 'contentful-sdk-core'
+import {
+  CreateAppSigningSecretProps,
+  AppSigningSecretProps,
+} from '../../../entities/app-signing-secret'
+import * as raw from './raw'
+import { RestEndpoint } from '../types'
+import { GetAppDefinitionParams } from '../../../common-types'
+
+export const get: RestEndpoint<'AppSigningSecret', 'get'> = (
+  http: AxiosInstance,
+  params: GetAppDefinitionParams
+) => {
+  return raw.get<AppSigningSecretProps>(
+    http,
+    `/organizations/${params.organizationId}/app_definitions/${params.appDefinitionId}/signing_secret`
+  )
+}
+
+export const create: RestEndpoint<'AppSigningSecret', 'create'> = (
```

Comment:
Why having two identical endpoints? (`create` and `update`)? There is no difference, right?
Should they maybe be combined? Something similar to `upsert` maybe.

Reply:
yeah, true. I didn't notice that we have this 'upsert' pattern - I'll change to that.